### PR TITLE
Added additional information to Data Collector Page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Data Collector Page

## Affected documentation pages

https://docs.magento.com/user-guide/system/support-data-collector.html

### CODE REVIEW

File: src/system/support-data-collector.md [hunk 1]
```diff
@@ -35,3 +35,25 @@ _Data Collector_
...
+
+1. Make sure it has _Complete_ status.
+
+1. Click the entity's name in _Code Dump_ or _DB Dump_ columns.
+
+The downloading to your computer should be started automatically.
+
+## Delete backup data
+
+1. On the _Admin_ sidebar, to **System** > _Support_ > **Data Collector**.
```
Comment (leslietilling) [inline src/system/support-data-collector.md:53]:
```suggestion
1. On the _Admin_ sidebar, go to **System** > _Support_ > **Data Collector**.
```

**Comment by DmitryTsymbal** [comment]
Hello @leslietilling and thank you for the review. I have applied requested changes, please check.
Thanks!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**